backend/app/migrator/migrator.go

```diff
@@ -24,6 +24,16 @@ type Exporter interface {
 	Export(w io.Writer, siteID string) (int, error)
 }
 
+// Mapper defines interface to convert data in import procedure
+type Mapper interface {
+	URL(url string) string
+}
+
+// NewMapper defines function that reads rules from reader and
+// returns new Mapper with loaded rules. If rules are not valid
+// it returns error.
+type NewMapper func(reader io.Reader) (Mapper, error)
```

Comment:
Same as in another comment - naming it MapperMaker looks nicer to me. However I would be fine if you decide to keep it, just a suggestion.